ckanext/canada/tables/choices/classification_group.yaml

```diff
@@ -29,181 +67,482 @@ CT-FIN:
-SO-MAO: {"en": "Ships' Officers", "fr": "Officiers de navire"}
-SO-RAD: {"en": "Ships' Officers", "fr": "Officiers de navire"}
-SR-APA: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-APB: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-BOB: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-CPS: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-EEW: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-ELE: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-EME: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-INM: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-MAC: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-MAM: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-MAN: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-MAT: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-MDO: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-MGT: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-MOC: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-PIP: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-PLE: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-PRW: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-QCW: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-SMW: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-SPS: {"en": "Ship Repair", "fr": "Réparation des navires"}
-SR-WOW: {"en": "Ship Repair", "fr": "Réparation des navires"}
-ST-COR: {"en": "Secretarial, Stenographic and Typing", "fr": "Secrétariat, sténographie et dactylographie"}
-ST-DNT: {"en": "Secretarial, Stenographic and Typing", "fr": "Secrétariat, sténographie et dactylographie"}
-ST-OCE: {"en": "Secretarial, Stenographic and Typing", "fr": "Secrétariat, sténographie et dactylographie"}
-ST-SCY: {"en": "Secretarial, Stenographic and Typing", "fr": "Secrétariat, sténographie et dactylographie"}
-ST-STN: {"en": "Secretarial, Stenographic and Typing", "fr": "Secrétariat, sténographie et dactylographie"}
-ST-TYP: {"en": "Secretarial, Stenographic and Typing", "fr": "Secrétariat, sténographie et dactylographie"}
-SW-CHA: {"en": "Social Work", "fr": "Service social"}
-SW-SCW: {"en": "Social Work", "fr": "Service social"}
-TI: {"en": "Technical Inspection", "fr": "Inspection technique"}
-TR: {"en": "Translation", "fr": "Traduction"}
-UT: {"en": "University Teaching", "fr": "Enseignement universitaire"}
-VM: {"en": "Veterinary Medicine", "fr": "Médecine vétérinaire"}
-WP: {"en": "Welfare Programmes", "fr": "Programmes de bien-être social"}
+SO-FLP:
+  en: Ships' Officers - Floating Plant
+  fr: Officiers de navire - Installations flottantes
+SO-INS:
+  en: Ships' Officers - Instructor
+  fr: Officiers de navire - Instructeur
+SO-MAO:
+  en: Ships' Officers - Marine Operations
+  fr: Officiers de navire - Navigation maritime
+SO-RAD:
+  en: Ships' Officers - Radio
+  fr: Officiers de navire - Radio
+SR-APA:  # 4th year
```

Comment:
should the year be in the label?

Reply:
Ah yeah these were from an archived classification. Got the labels and translations in now.